@ecomplus/storefront-app/src/components/js/EcCheckout.js

```diff
@@ -290,8 +290,8 @@ export default {
 
     selectAddress (addressId) {
       this.$emit('address-selected', addressId)
-      this.updateZipCode()
       this.selectedAddressId = addressId
+      this.updateZipCode()
```

Comment:
Só isso aí provavelmente não vai adiantar não 🙃

Reply:
Na verdade isso aí não muda praticamente nada, os processos ainda são assíncronos, por isso o callback `this.$nextTick(this.updateZipCode)`

Reply:
Tinha funcionado, mas coloco isso